packages/file_selector/file_selector_linux/linux/test/file_selector_plugin_test.cc

```diff
@@ -135,14 +163,13 @@ TEST(FileSelectorPlugin, TestSaveSimple) {
 }
 
 TEST(FileSelectorPlugin, TestSaveWithArguments) {
-  g_autoptr(FlValue) args = fl_value_new_map();
-  fl_value_set_string_take(args, "initialDirectory",
-                           fl_value_new_string("/tmp"));
-  fl_value_set_string_take(args, "suggestedName",
-                           fl_value_new_string("foo.txt"));
+  g_autoptr(FfsPlatformFileChooserOptions) options =
+      ffs_platform_file_chooser_options_new(nullptr, "initialDirectory",
```

Comment:
Should this be "/tmp"?

Reply:
Oops, yes. I made this mistake with both arguments, but when the test caught one I didn't notice that I'd done it both times (and the test didn't catch the other because of the TODO further down). Fixed!